src/help/components/results/Results.tsx

```diff
@@ -117,6 +118,9 @@ const Results = ({ history, location }: RouteChildrenProps) => {
         </>
       }
     >
+      <Helmet>
+        <title>{searchValue} in UniProt help</title>
```

Comment:
I'll comment that in there, because I'm not sure of the help URLs will evolve as we're actively working on it now. So maybe it will need another approach altogether.